src/wram.asm

```diff
@@ -1406,10 +1431,10 @@ wceaa:: ; ceaa
 wceab:: ; ceab
 	ds $4
 
-wceaf:: ; ceaf
+wCursorDuelXPosition:: ; ceaf
```

Comment:
I feel these two are also specific enough to the check menu and submenus that I would try to reflect it. For example, what do you think of ``CheckMenuCursorXPosition`` and ``CheckMenuCursorYPosition``?